### PR TITLE
TELCODOCS-1689 - Adding PolicyGenerator release notes

### PR DESCRIPTION
**Pull request by aireilly**
Recommend ACM PolicyGenerator over ztp PolicyGenTemplate for managed cluster polices release notes

Version(s):
enterprise-4.16

Issue:
https://issues.redhat.com/browse/TELCODOCS-1689

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Comment by aireilly** [comment]
Note PR will fail until https://github.com/openshift/openshift-docs/pull/76691 is merged

**Comment by mletalie** [comment]
> Note PR will fail until #76691 is merged

Thanks for the heads up, was about to reach out to you :) . Appreciate it!

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -364,6 +364,15 @@ With this release, you can increase the disk quota in etcd. This is a Technology
...
 === Edge computing
 
+[id="ocp-4-16-edge-computing_pg-ztp-rhacm_{context}"]
+==== Using {rh-rhacm} PolicyGenerator resources to manage {ztp} cluster policies
+
+You can now use `PolicyGenerator` resources and {rh-rhacm-first} to deploy polices for managed clusters with {ztp}.
+The `PolicyGenerator` API is part of the link:https://open-cluster-management.io/[Open Cluster Management] standard and provides a generic way of patching resources which is not possible with the `PolicyGenTemplate` API.
+Using `PolicyGenTemplate` resources to manage and deploy polices will be deprecated in an upcoming OpenShift Container Platform release.
+
+For more infromation, see xref:../edge_computing/policygenerator_for_ztp/ztp-configuring-managed-clusters-policygenerator[Configuring managed cluster policies by using PolicyGenerator resources].
```
Comment (mletalie) [inline release_notes/ocp-4-16-release-notes.adoc:374]:
```suggestion
For more information, see xref:../edge_computing/policygenerator_for_ztp/ztp-configuring-managed-clusters-policygenerator[Configuring managed cluster policies by using PolicyGenerator resources].
```

Comment (mletalie) [reply release_notes/ocp-4-16-release-notes.adoc:374]: Not sure if link is correct given the target? May want to double check.

**Review by mletalie — COMMENTED** [review]
/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/lgtm

**Comment by mletalie** [comment]
Hello @aireilly, just one spelling issue and a comment. Otherwise, lgtm. Good job!

**Review by edcdavid — APPROVED** [review]

**Comment by josclark42** [comment]
lgtm from QE

**Pull request merged**